flink-end-to-end-tests/flink-end-to-end-tests-common-kafka/pom.xml

```diff
@@ -49,8 +49,15 @@ under the License.
 			<scope>compile</scope>
 		</dependency>
 		<dependency>
-			<groupId>junit</groupId>
-			<artifactId>junit</artifactId>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter</artifactId>
+			<scope>compile</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-migrationsupport</artifactId>
```

Comment:
Do we even need this? Afaik this is for using JUnit4 rules in JUnit5 tests. 
I had expected instead to see the vintage runner.

Same comment for the other modules.